### PR TITLE
interim check in for 176099

### PR DESCRIPTION
**Pull request by eklein-okta**
## Description:
-Added /api/v1/authn/info (GET) 

### Resolves:

*[OKTA-176099](https://oktainc.atlassian.net/browse/OKTA-176099)

### CODE REVIEW

File: _source/_docs/api/resources/authn.md [hunk 1]
```diff
@@ -6371,6 +6371,116 @@ curl -v -X POST \
...
+
+##### Request Parameters for Get Information
+{:.api .api-request .api-request-params}
+
+None
+
+##### Response Parameters for Get Information
+{:.api .api-response .api-response-params}
+
+As part of the authentication call either the username and password or the token parameter must be provided.
```
Comment (edjohns-okta) [inline _source/_docs/api/resources/authn.md]: Its an anonymous request.  Neither a token nor a username\password need be provided.

File: _source/_docs/api/resources/authn.md [hunk 2]
```diff
@@ -6371,6 +6371,116 @@ curl -v -X POST \
...
 
+## Miscellaneous Operations
+
+### Get Information
+{:.api .api-operation}
+
+{% api_operation get /api/v1/authn/info %}
+
+
+Fetches general information
```
Comment (edjohns-okta) [inline _source/_docs/api/resources/authn.md]: Maybe say "Fetches general system information" or "Fetches general organization information"?

File: _source/_docs/api/resources/authn.md [hunk 3]
```diff
@@ -6371,6 +6371,116 @@ curl -v -X POST \
...
+| name    | Company name s | Body | String   | FALSE    |          |
+| technicalContact | Company technical contact | Body | String   | FALSE    |          |
+| _links        | [link relations](#links-object) for the current `status`                                               | [JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06) | TRUE     | TRUE     |           |
+
+
+##### Request Example for Get Information
+{:.api .api-request .api-request-example}
+
+~~~sh
+curl -v -X POST \
```
Comment (edjohns-okta) [inline _source/_docs/api/resources/authn.md]: Its a GET not a POST

File: _source/_docs/api/resources/authn.md [hunk 4]
```diff
@@ -6371,6 +6371,116 @@ curl -v -X POST \
...
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
```
Comment (edjohns-okta) [inline _source/_docs/api/resources/authn.md]: An API token is not required.

File: _source/_docs/api/resources/authn.md [hunk 5]
```diff
@@ -6371,6 +6371,116 @@ curl -v -X POST \
...
+"{yourOktaDomain}/api/v1/usersauthn/info"
```
Comment (edjohns-okta) [inline _source/_docs/api/resources/authn.md]: The form of the url is: {yourOktaDomain}/api/v1/authn/info

File: _source/_docs/api/resources/authn.md [hunk 6]
```diff
@@ -6371,6 +6371,116 @@ curl -v -X POST \
...
+~~~
+
+##### Response Example for Get Information
+{:.api .api-response .api-response-example}
+
+~~~json
+{
+   “name”: “My Company Name”,
+   “technicalContact”: “tech_contact@mycompany.com”,
```
Comment (edjohns-okta) [inline _source/_docs/api/resources/authn.md]: The support phone number (if present) will also be returned: 
     "name": "FMG Software Inc",
    "supportPhoneNumber": "253 223 6986",
    "technicalContact": "ed.johns@okta.com",

File: _source/_docs/api/resources/authn.md [hunk 7]
```diff
@@ -6371,6 +6371,116 @@ curl -v -X POST \
...
+
+##### Response Parameters for Get Information
+{:.api .api-response .api-response-params}
+
+As part of the authentication call either the username and password or the token parameter must be provided.
+
+| Parameter   | Description                                                                                                            | Param Type | DataType                          | Required | MaxLength
+| ----------- | :--------------------------------------------------------------------------------------------------------------------- | :--------- | :-------------------------------- | :------- | :------- |
+| name    | Company name s | Body | String   | FALSE    |          |
+| technicalContact | Company technical contact | Body | String   | FALSE    |          |
```
Comment (edjohns-okta) [inline _source/_docs/api/resources/authn.md]: Add the support phone number here as well (see below)

File: _source/_docs/api/resources/authn.md [hunk 8]
```diff
@@ -6371,6 +6371,116 @@ curl -v -X POST \
...
+-H "Authorization: SSWS ${api_token}" \
+"{yourOktaDomain}/api/v1/usersauthn/info"
+~~~
+
+##### Response Example for Get Information
+{:.api .api-response .api-response-example}
+
+~~~json
+{
+   “name”: “My Company Name”,
```
Comment (jakubvul) [inline _source/_docs/api/resources/authn.md]: You must use `"` and not `“`, or GitHub will start this red highlighting.

File: _source/_docs/api/resources/authn.md (hunk 2 above)
Comment (jakubvul) [reply _source/_docs/api/resources/authn.md]: I don't find any of these three sentences to be informative enough to be helpful.

File: _source/_docs/api/resources/authn.md [hunk 9]
```diff
@@ -6371,6 +6371,116 @@ curl -v -X POST \
...
+   “technicalContact”: “tech_contact@mycompany.com”,
+   “_links”: {
+       “login”: [
+           {
+               “href”: “https://mycompany.okta.com:80“,
```
Comment (jakubvul) [inline _source/_docs/api/resources/authn.md]: Please replace `https://mycompany.okta.com:80` with `https://{yourOktaDomain}` throughout.

**Comment by eklein-okta** [comment]
@edjohns-okta, @jakubvul Edits from the first review are entered.

File: _source/_docs/api/resources/authn.md [hunk 10]
```diff
@@ -6371,6 +6371,117 @@ curl -v -X POST \
...
+"{yourOktaDomain}/api/v1/authn/info"
+~~~
+
+##### Response Example for Get Information
+{:.api .api-response .api-response-example}
+
+~~~json
+{
+   "name": "My Company Name",
+   "supportPhoneNumber": "415 555 6582"
```
Comment (jakubvul) [inline _source/_docs/api/resources/authn.md]: Missing a comma. Is there a reason we're not just copying and pasting an actual JSON response?

Comment (eklein-okta) [reply _source/_docs/api/resources/authn.md]: Technical difficulties with the writer's setup...on the list...

File: _source/_docs/api/resources/authn.md (hunk 7 above)
Comment (edjohns-okta) [reply _source/_docs/api/resources/authn.md]: Should it be company name or organization name?

**Comment by bretterer** [comment]
Closing this PR as stale.  If you would like to make the above changes, please feel free to re-open this PR

**Pull request closed without merging**